lib/ruy/conditions/assert.rb

```diff
@@ -1,20 +1,30 @@
 module Ruy
   module Conditions
 
-    # Asserts that an attribute has a truthy value.
+    # Expects that the context key stores a truthy value
+    #
     class Assert < Condition
-      attr_reader :attr
+      attr_reader :key
 
-      # @param attr Context attribute's attr
-      def initialize(*attrs)
+      # @param key
+      # @example evaluate that :key passes
+      #   Asset.new(:key)
+      def initialize(*keys)
         super
-        @attr = attrs.first if attrs.any?
+        @key = keys.first if keys.any?
```

Comment:
This was done on purpose to allow `nil` as a value for a key. `has_key?` checks whether `@key` is defined or not. Setting it to `nil` will make `defined?` return `true` which is not the expected outcome.